### PR TITLE
[Flutter Parent][MBL-13021] Message instructor from assignment details

### PR DESCRIPTION
**Pull request by StephenBrough**
For testing:

* Subject should be 'Regarding [student name], Assignment - [Assignment name]'

* There should be a link to the assignment appended after send (the user won't see it in the message body until after they have sent the message).

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1020086&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1020087&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1020088&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

File: apps/flutter_parent/lib/screens/assignments/assignment_details_screen.dart [hunk 1]
```diff
@@ -96,11 +108,16 @@ class _AssignmentDetailsScreenState extends State<AssignmentDetailsScreen> {
...
 
-  Widget _fab() => FloatingActionButton(
-        onPressed: _sendMessage,
-        tooltip: L10n(context).assignmentMessageHint,
-        child: Padding(padding: const EdgeInsets.only(left: 4, top: 4), child: Icon(CanvasIconsSolid.comment)),
-      );
+  Widget _fab(AsyncSnapshot<AssignmentDetails> snapshot) {
+    print('snapshot: $snapshot');
+    print('snapshot data: ${snapshot.data}');
+    print('snapshot assignment: ${snapshot.data.assignment}');
```
Comment (JordanMarshall) [inline apps/flutter_parent/lib/screens/assignments/assignment_details_screen.dart]: Are these prints needed?

Comment (StephenBrough) [reply apps/flutter_parent/lib/screens/assignments/assignment_details_screen.dart]: LOL

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1020107&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Pull request merged**